charts/kyverno/templates/_helpers.tpl

```diff
@@ -116,3 +116,30 @@ maxUnavailable: {{ .Values.podDisruptionBudget.maxUnavailable }}
 {{- define "kyverno.imagePullSecret" }}
 {{- printf "{\"auths\":{\"%s\":{\"auth\":\"%s\"}}}" .registry (printf "%s:%s" .username .password | b64enc) | b64enc }}
 {{- end }}
+
+
+{{- define "kyverno.resourceFilters" -}}
+{{- $resourceFilters := .Values.config.resourceFilters }}
+{{- if .Values.excludeKyvernoNamespace }}
+  {{- $resourceFilters = prepend .Values.config.resourceFilters (printf "[*,%s,*]" (include "kyverno.namespace" .)) }}
+{{- end }}
+{{- tpl (join "" $resourceFilters) . }}
+{{- end }}
+
+{{- define "kyverno.webhooks" -}}
+{{- $excludeDefault := dict "key" "kubernetes.io/metadata.name" "operator" "NotIn" "values" (list (include "kyverno.namespace" .)) }}
+{{- $newWebhook := list }}
+{{- range $webhook := .Values.config.webhooks }}
+  {{- $namespaceSelector := default dict $webhook.namespaceSelector }}
+  {{- $matchExpressions := default list $namespaceSelector.matchExpressions }}
+  {{- if $.Values.excludeKyvernoNamespace }}
+    {{- $namespaceSelector = dict "matchLabels" $namespaceSelector.matchLabels "matchExpressions" (append $matchExpressions $excludeDefault) }}
+  {{- end }}
+  {{- if (empty (omit $webhook "namespaceSelector")) }}
+    {{- $newWebhook = list (dict "namespaceSelector" $namespaceSelector)}}
+  {{- else }}
+    {{- $newWebhook = list (omit $webhook "namespaceSelector") (dict "namespaceSelector" $namespaceSelector) }}
+  {{- end }}
```

Comment:
This doesn't look correct to me.

Reply:
It wasn't correct.  The thing that's needed is to append to `$newWebhook` the new values. I fixed this and condensed logic a bit.